internal/config/project_context.go

```diff
@@ -85,13 +86,36 @@ func DetectProjectMetadata(path string) *schema.ProjectMetadata {
 
 	vcsRepoURL = stripVCSRepoPassword(vcsRepoURL)
 
-	return &schema.ProjectMetadata{
+	meta, err := vcs.GetMetadata()
```

Comment:
If I run `infracost breakdown --path=../other/path` in the directory with CLI source code, it picks up metadata of the CLI git repo. If we pass project's path here, we will be able to detect local git settings of the project.

Reply:
yeah probably a good shout

Reply:
done in https://github.com/infracost/infracost/pull/1758/commits/d7efcc9de67daf36141b352847ff67a2355e00eb